content/docs/0.6.0/usecases/quality-gates/index.md

```diff
@@ -0,0 +1,197 @@
+    ```
+
+    ```console
+    cd examples/onboarding-carts
+    ```
+
+## Install Keptn just for this use case
+
+If you want to install Keptn just to explore the capabilities of Keptn quality gates, you have the option to roll-out Keptn **without** components for automated delivery and operations. Therefore, set the `use-case` flag to `quality-gates` when executing the [install](../../reference/cli/#keptn-install) command as shown below:
+
+```console
+keptn install --platform=[aks|eks|gke|openshift|pks|kubernetes] --use-case=quality-gates
+```
+
+## Configure Keptn
+
+If you have completed another tutorial with the sockshop project and carts service, you can either delete the project using the [delete project](../../reference/cli/#keptn-delete-project) command or you use another project name throughout this tutorial, e.g., *rockshop*.
+
+* Create the Keptn project *sockshop* with only one the *hardening* stage declared in the `shipyard_quality_gates.yaml` file:
+
+  ```
+  keptn create project sockshop --shipyard=shipyard_quality_gates.yaml
+  ```
+
+* Create the Keptn service *carts* that you want to evaluate using Keptn quality gates:
+
+  ```console
+  keptn create service carts --project=sockshop
+  ```
+
+## Activate quality gate and monitoring
+
+* Activate the quality gate for the carts service. Therefore, upload the `slo_quality-gates.yaml` file using the [add-resource](../../reference/cli/#keptn-add-resource) command:
+
+  ```console
+  keptn add-resource --project=sockshop --service=carts --stage=hardening --resource=slo_quality-gates.yaml --resourceUri=slo.yaml
+  ```
+
+For this tutorial you will need to set up monitoring, either using the open-source monitoring solution *Prometheus* or *Dynatrace*. 
+
+### Option 1: Prometheus
+<details><summary>Expand instructions</summary>
+<p>
+
+1. Configure Prometheus monitoring for the **sockshop** project and **carts** service as explained [here](../../reference/monitoring/prometheus/#setup-prometheus).
+
+1. Configure the Prometheus SLI provider for the **sockshop** project as explained [here](../../reference/monitoring/prometheus/#setup-prometheus-sli-provider). The ConfigMap that need to be applied is provided in the `examples/onboarding-carts` folder.
+
+1. To configure Keptn to use the Prometheus SLI provider for the **sockshop** project, apply the below ConfigMap by executing the following command from within the `examples/onboarding-carts` folder:
+
+    ```console
+    kubectl apply -f lighthouse-source-prometheus.yaml
+    ```
+
+    ```yaml
+    apiVersion: v1
+    data:
+      sli-provider: prometheus
+    kind: ConfigMap
+    metadata:
+      name: lighthouse-config-sockshop
+      namespace: keptn
+    ```
+
+</p>
+</details>
+
+### Option 2: Dynatrace
+<details><summary>Expand instructions</summary>
+<p>
+
+1. Please complete the instructions for setting up [Dynatrace monitoring](../../reference/monitoring/dynatrace#setup-dynatrace).
```

Comment:
Why does one have to setup Dynatrace (or Prometheus in Option 1)? For the quality gates standalone we only need the dynatrace-sli-service (or the prometheus-sli-service) and need to configure it.

Reply:
Right, is not necessary. I removed the Dynatrace (or Prometheus) setup and just point to deploying the SLI provider.